### PR TITLE
Add customization settings 

### PR DESCRIPTION
**Pull request by Quetzacoalt91**
This PR fixes the issue [BOOM-767](http://forge.prestashop.com/browse/BOOM-767) and is made for the Europe compliance.

* Number of days the products are reserved
* Custom text to display on the bankwire presentation

### CODE REVIEW

File: bankwire.php [hunk 1]
```diff
@@ -298,15 +343,27 @@ public function renderForm()
...
 
     public function getConfigFieldsValues()
     {
+        $custom_text = array();
+        $languages = Language::getLanguages(false);
+        foreach ($languages as $lang) {
+            $custom_text[$lang['id_lang']] = Tools::getValue(
+                'BANK_WIRE_CUSTOM_TEXT_'.$lang['id_lang'],
+                Configuration::get('BANK_WIRE_CUSTOM_TEXT', $lang['id_lang']
+                )
```
Comment (maximebiloe) [inline bankwire.php]: Can you put the closing parenthesis on the previous line? 😄

File: views/templates/hook/payment_return.tpl [hunk 2]
```diff
@@ -24,22 +24,16 @@
...
-			<dt>{l s='Name of account owner' mod='bankwire'}</dt>
-			<dd>{$bankwireOwner}</dd>
-			<dt>{l s='Please include these details' mod='bankwire'}</dt>
-			<dd>{$bankwireDetails}</dd>
-			<dt>{l s='Bank name' mod='bankwire'}</dt>
-			<dd>{$bankwireAddress}</dd>
-			<dt>{l s='Order reference' mod='bankwire'}</dt>
-			<dd>{$reference}</dd>
-		</dl>
+	<p>
```
Comment (maximebiloe) [inline views/templates/hook/payment_return.tpl]: Can you check the indent, please?

File: views/templates/hook/bankwire_intro.tpl [hunk 3]
```diff
@@ -0,0 +1,52 @@
...
+* versions in the future. If you wish to customize PrestaShop for your
+* needs please refer to http://www.prestashop.com for more information.
+*
+*  @author PrestaShop SA <contact@prestashop.com>
+*  @copyright  2007-2015 PrestaShop SA
+*  @license    http://opensource.org/licenses/afl-3.0.php  Academic Free License (AFL 3.0)
+*  International Registered Trademark & Property of PrestaShop SA
+*}
+
+<section>
```
Comment (maximebiloe) [inline views/templates/hook/bankwire_intro.tpl]: Can you uniform the indent with two spaces, please ?

File: bankwire.php (hunk 1 above)
Comment (Quetzacoalt91) [inline bankwire.php]: https://github.com/PrestaShop/ps_emailsubscription/commit/ecffc8e2c1dd8763d8d7d313a0ecffd7dfb2dff6#diff-cfb1aa2304713070e111aaa57e5c8ef5R1031

# Do as I say, not as I do :laughing:

Comment (maximebiloe) [inline bankwire.php]: my-bad

**Comment by Quetzacoalt91** [comment]
Changes have been applied

**Comment by dSkrbic** [comment]
QA Approved :)

**Pull request merged**